### PR TITLE
API-11015: splitted declarations for Customer and Agent API

### PR DESCRIPTION
**Pull request by grz-gajda**

### CODE REVIEW

File: src/objects/index.ts [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+  Agent as CustomerAgent,
+  Statistics as CustomerStatistics,
+  Geolocation as CustomerGeolocation,
+  LastPage as CustomerLastPage,
+  Visit as CustomerVisit,
+ } from "../agent/structures";
+ import {
+  Customer as AgentCustomer,
+  Agent as AgentAgent,
+ } from "../customer/structures";
```
Comment (olek-p) [inline src/objects/index.ts:12]: Shouldn't these names be reversed? I mean, import `CustomerX` from `../customer/structures` and not from `../agent/structures`

File: src/objects/index.ts [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+
+export type Agent = AgentAgent | AgentCustomer;
```
Comment (olek-p) [inline src/objects/index.ts:14]: Also, shouldn't this be
```suggestion
export type Agent = AgentAgent | CustomerAgent;
```
?

Comment (grz-gajda) [reply src/objects/index.ts]: https://github.com/livechat/lc-sdk-js/pull/65#discussion_r907209380

File: src/objects/index.ts (hunk 1 above)
Comment (olek-p) [reply src/objects/index.ts]: That will do

**Review by olek-p — APPROVED** [review]

**Pull request merged**